### PR TITLE
Feature/reconsent signup screen

### PR DESCRIPTION
**Pull request by aikewoody**
# Description

The connection to the backend is not implemented yet but will be done with a different ticket

https://www.notion.so/joinzoe/017c3de591604a7b89d0b69e3c37b279?v=0133357df3ed42e1bd42c1e0d7afcf45&p=060373e071d94a488377db416fc60eb8

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

Simulator Screen Shot - iPhone 11 - 2021-07-02 at 16 39 38

Simulator Screen Shot - iPhone 11 - 2021-07-02 at 16 39 41

## Checklist (for submission)

- [ ] Analytics/tracking events have been added (if needed)

## Checklist (for reviewers)

- [x] Checked against Figma designs (if relevant)
- [ ] Checked that data has been successfully saved to the backend (if relevant)

## Out of scope and potential follow-up

_Are there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?_

### CODE REVIEW

File: src/features/reconsent/screens/ReconsentNewsletterSignupScreen.tsx [hunk 1]
```diff
@@ -1,31 +1,100 @@
...
+  top: 24,
+};
 
 export default function ReconsentNewsletterSignupScreen() {
-  const theme = useTheme();
+  const [loading, setLoading] = React.useState(false);
+  const [signedUp, setSignedUp] = React.useState(false);
+
+  function toggleNewsletterSignup() {
+    setSignedUp(!signedUp);
```
Comment (juliatan) [inline src/features/reconsent/screens/ReconsentNewsletterSignupScreen.tsx:24]: Hi, so my understanding with React is that it's safer to reference previous state explicitly if new state relies on previous state (https://reactjs.org/docs/hooks-reference.html#functional-updates).
i.e. `setSignedup((prevState) => !prevState)`

File: src/features/reconsent/components/ReconsentScreen.tsx [hunk 2]
```diff
@@ -22,22 +22,21 @@ const dots = Array(AMOUNT_DOTS)
   .fill(null)
   .map((_, i) => i);
 
+const hitSlop = {
```
Comment (juliatan) [inline src/features/reconsent/components/ReconsentScreen.tsx:25]: Please can you remove this from your PR? I've been making a bunch of changes to this file and don't want to deal with the merge conflicts again. :) I'll add this changes in my commit.

**Review by juliatan — APPROVED** [review]
👍🏻

File: src/features/reconsent/screens/ReconsentNewsletterSignupScreen.tsx (hunk 1 above)
Comment (aikewoody) [reply src/features/reconsent/screens/ReconsentNewsletterSignupScreen.tsx:24]: Good point, this function is dummy btw until the api is present for doing the actual signup

**Pull request merged**